securedrop/tests/test_crypto_util.py

```diff
@@ -109,24 +109,6 @@ def test_encrypt_binary_stream(source_app, config, test_source):
         assert fh.read() == plaintext
 
 
-def test_encrypt_fingerprints_not_a_list_or_tuple(source_app, test_source):
```

Comment:
Not needed anymore since encrypt() always takes a list in this PR.

Reply:
:+1: